.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
This is essentially not required at all, as the prettier does the job. You just need to enable it in your editor. I would rather add a suggestion for adding a prettier plugin for vs code (esbenp.prettier-vscode)